### ISSUE TITLE
Variables into if Statement

### ISSUE THREAD

**Issue by lavaldi** [galenframework/galen#312]
Hi Ivan, 
I want to put a variable in the if statement but I get an error.

This is my code:
```
        @for [1 - 35] as index
            @if ${isVisible("clients-slider-list-${index}")}
                clients-slider-list-${index}:
                    left-of clients-slider-list-${index + 1} 0px
                    aligned horizontally all clients-slider-list-${index + 1}
                    inside clients-slider-list 0px top bottom
                    component components/client_item.gspec
```
This is the error:
![Image of Error](http://i.imgur.com/DOKPYh6.png)

**Comment by ishubin** [comment]
This will not work as the ```${...}``` is a galen syntax for executing the JavaScript. And it will not work within the JavaScript itself. I understand what you want to achieve. Basically you should be able to do it like this:

```
 @for [1 - 35] as index
            @if ${isVisible("clients-slider-list-" + index)}
                clients-slider-list-${index}:
```

Unfortunately the code above does not work due to bug I've just found. I have already pushed the fix to release-2.1 branch. It will be soon released (in a week or two). For now you can use a workaround:

```
 @for [1 - 35] as index
            @if ${isVisible(("clients-slider-list-" + index).toString())}
                clients-slider-list-${index}:
```

It is ugly, I know but once the version 2.1 is live you will be able to get rid of this ```.toString()``` function and it should work

**Comment by lavaldi** [comment]
Thanks Ivan ^_^
